docs/src/release_notes.md

```diff
@@ -2,6 +2,7 @@
 
 History of user-visible changes for CloudNativePG, classified for each minor release.
 
+- [CloudNativePG 1.19](release_notes/v1.18.md)
```

Comment:
this shouldn't be `release_notes/v1.19.md` ?

Reply:
thank you, sir